pom.xml

```diff
@@ -522,9 +480,19 @@
                 <version>${mockito.version}</version>
                 <scope>test</scope>
             </dependency>
+
+            <dependency>
```

Comment:
Should be removed, was only added for jaxrs plugin, which is archived now.

Reply:
Do you refer to the Mockito dependency?